library/src/main/java/com/novoda/downloadmanager/FileNameExtractor.java

```diff
@@ -0,0 +1,21 @@
+package com.novoda.downloadmanager;
+
+final class FileNameExtractor {
+
+    private static final String PATH_SEPARATOR = "/";
```

Comment:
Why not using just `File.pathSeparatorChar`?

Reply:
I didn't know this existed tbh but now that I do I can say that it won't work 😄  From the docs:

```
     * The system-dependent path-separator character.  This field is
     * initialized to contain the first character of the value of the system
     * property <code>path.separator</code>.  This character is used to
     * separate filenames in a sequence of files given as a <em>path list</em>.
     * On UNIX systems, this character is <code>':'</code>; on Microsoft Windows systems it
     * is <code>';'</code>.
```

It's using the filesystem separator, we want the network address separator.

Reply:
AH right, fair enough 👍

Reply:
should we rename this constant to  network address separator?